[2023-03-20T14:52:23.919+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: project_transform_bq_data.create_dataproc_cluster manual__2023-03-20T00:12:34.684738+00:00 [queued]>
[2023-03-20T14:52:23.938+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: project_transform_bq_data.create_dataproc_cluster manual__2023-03-20T00:12:34.684738+00:00 [queued]>
[2023-03-20T14:52:23.939+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-20T14:52:23.940+0000] {taskinstance.py:1283} INFO - Starting attempt 16 of 17
[2023-03-20T14:52:23.940+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-20T14:52:23.974+0000] {taskinstance.py:1303} INFO - Executing <Task(DataprocCreateClusterOperator): create_dataproc_cluster> on 2023-03-20 00:12:34.684738+00:00
[2023-03-20T14:52:23.985+0000] {standard_task_runner.py:55} INFO - Started process 3208 to run task
[2023-03-20T14:52:23.998+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'project_transform_bq_data', 'create_dataproc_cluster', 'manual__2023-03-20T00:12:34.684738+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/p_spark_processing.py', '--cfg-path', '/tmp/tmpgjyq1ftx']
[2023-03-20T14:52:24.002+0000] {standard_task_runner.py:83} INFO - Job 198: Subtask create_dataproc_cluster
[2023-03-20T14:52:24.031+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:250: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-03-20T14:52:24.143+0000] {task_command.py:388} INFO - Running <TaskInstance: project_transform_bq_data.create_dataproc_cluster manual__2023-03-20T00:12:34.684738+00:00 [running]> on host 4fd00a82f4e9
[2023-03-20T14:52:24.324+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_transform_bq_data
AIRFLOW_CTX_TASK_ID=create_dataproc_cluster
AIRFLOW_CTX_EXECUTION_DATE=2023-03-20T00:12:34.684738+00:00
AIRFLOW_CTX_TRY_NUMBER=16
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-20T00:12:34.684738+00:00
[2023-03-20T14:52:24.326+0000] {dataproc.py:601} INFO - Creating cluster: my-dataproc-cluster
[2023-03-20T14:52:24.328+0000] {crypto.py:83} WARNING - empty cryptography key - values will not be stored encrypted.
[2023-03-20T14:52:24.329+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-03-20T14:55:38.526+0000] {dataproc.py:614} INFO - Cluster created.
[2023-03-20T14:55:38.648+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=project_transform_bq_data, task_id=create_dataproc_cluster, execution_date=20230320T001234, start_date=20230320T145223, end_date=20230320T145538
[2023-03-20T14:55:38.812+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-20T14:55:38.925+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
